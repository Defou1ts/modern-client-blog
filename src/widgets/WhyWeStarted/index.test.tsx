import React from 'react';

import renderer from 'react-test-renderer';

import { WhyWeStarted } from './index';

describe('WhyWeStarted', () => {
	it('WhyWeStarted snapshot', () => {
		const component = renderer.create(<WhyWeStarted />);

		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
