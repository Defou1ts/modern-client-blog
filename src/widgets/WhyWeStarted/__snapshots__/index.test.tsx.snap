// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WhyWeStarted WhyWeStarted snapshot 1`] = `
<article
  className="wrapper"
>
  <div
    className="backgroundWrapper"
  />
  <div
    className="content"
  >
    <p
      className="subtitle"
    >
      why-we-started.subtitle
    </p>
    <h1
      className="h h1 title"
    >
      why-we-started.title
    </h1>
    <p
      className="p medium description"
    >
      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip.
    </p>
    <a
      href="/about"
      onClick={[Function]}
      onMouseEnter={[Function]}
      onTouchStart={[Function]}
    >
      <button
        className="button primary button"
      >
        why-we-started.button
      </button>
    </a>
  </div>
</article>
`;
